keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -795,79 +840,149 @@ enhancement:
 * **Are there any tests for feature enablement/disablement?**
   The e2e framework does not currently support enabling and disabling feature
   gates. However, unit tests in each component dealing with managing data created
-  with and without the feature are necessary. At the very least, think about
-  conversion tests if API types are being modified.
+  with and without the feature are necessary and will be added before
+  before the transition to beta.
 
 ### Rollout, Upgrade and Rollback Planning
 
-Will be added before the transition to beta.
-
 * **How can a rollout fail? Can it impact already running workloads?**
 
+A rollout happens in at least two phases:
+1. Updating the cluster so that the `CSIStorageCapacity` API is enabled in the apiserver
+   and the kube-scheduler uses that information *for drivers which have opted into this*.
+2. CSI driver installations get updated such that they produce `CSIStorageCapacity` objects
+   and enable usage of those objects in their `CSIDriver` object.
+
+If for whatever reason the first phase fails, the feature can be left
+disabled and the second phase can be skipped. The cluster the operates
+as before.
+
+If the second phase fails because a driver malfunctions or overloads
+the apiserver, then it can be rolled back and scheduling again happens
+without using storage capacity information.
+
+In none of these cases are running workloads affected.
+
 * **What specific metrics should inform a rollback?**
 
+Increased number of pods that are not getting scheduled and apiserver
```

Comment:
"pods not being scheduled" is tricky - because there can be other reasons of that. Can we be more specific?

apiserver load - can we also be more specific? Just looking at CPU? Or maybe looking into api call latencies?
[The later seems probably better - please add the specific metric name]